flang/test/Semantics/OpenMP/name-conflict.f90

```diff
@@ -0,0 +1,24 @@
+!RUN: %python %S/../test_errors.py %s %flang_fc1 -fopenmp
+module m
+
+contains
+
+subroutine foo1()
+  integer :: baz1
+!ERROR: 'baz1' must be a variable
+!$omp parallel do shared(baz1)
+  baz1: do i = 1, 100
+  enddo baz1
+!$omp end parallel do
+end subroutine
+
+subroutine foo2()
+  !implicit baz2
```

Comment:
Nit : If you are not planning to use this commented implicit can you please remove it?

Reply:
It's there to say that `baz2` is implicit...